tests/integration/modules/test_git.py

```diff
@@ -56,7 +56,7 @@ def _worktrees_supported():
     Check if the git version is 2.5.0 or later
     '''
     try:
-        return _git_version() >= LooseVersion('2.5.0').encode()
+        return _git_version() >= LooseVersion('2.5.0')
```

Comment:
`_git_version()` returns `LooseVersion()` without the `.encode()`
The `LooseVersion()` class doesn't have an `encode()` method. Perhaps they meant to put it inside the parenthesis... (`LooseVersion('2.5.0'.encode())` I'll try that...